isacc_messaging/api/fhir.py

```diff
@@ -35,6 +36,51 @@ def resolve_reference(reference_string):
     raise IsaccNotFoundError("{reference_string} NOT FOUND")
 
 
+def first_in_bundle(bundle):
+    """Return first resource in bundle
+
+    :param bundle:  Fresh JSON bundle from FHIR store
+    :return: first resource found in bundle
+    """
+    if bundle['resourceType'] == 'Bundle':
+        if bundle['total'] > 0:
+            return bundle['entry'][0]['resource']
+
+
+def next_in_bundle(bundle):
+    """Generator to manage paging through bundle results
+
+    :param bundle:  Fresh JSON bundle from FHIR store
+
+    Yields each respective resource from the bundle's entry list
+    until exhousted.
+    """
+    def next_in_page(result):
+        if result['resourceType'] == 'Bundle' and result['total'] > 0:
+            for entry in result['entry']:
+                yield entry['resource']
+
+    def next_page(result):
+        while len([link['url'] for link in result["link"] if link['relation'] == 'next']) > 0:
+            next_page_url = [link['url'] for link in result["link"] if link['relation'] == 'next'][0]
+            next_page_url = urlsplit(next_page_url)
+            params = parse_qs(next_page_url.query)
+            result = HAPI_request('GET', '', params=params)
```

Comment:
Why override the default of `None` for `''`? 
```suggestion
            result = HAPI_request('GET', params=params)
```

Reply:
done by someone before me ;)